src/dymaptic.GeoBlazor.Core/Scripts/jsBuilder.ts

```diff
@@ -552,6 +558,116 @@ export function buildJsRenderer(dotNetRenderer: any): Renderer | null {
     return dotNetRenderer
 }
 
+//export function buildJsRasterColormapRenderer(dotNetRasterColormapRenderer: DotNetRasterColormapRenderer): RasterColormapRenderer | null {
```

Comment:
why is this all commented out?

Reply:
This code was commented out while working on the one imagery layer renderer, prior to testing to the other renderers.  I believe this was an oversight in un commenting this code prior to pushing the code up.

Code is now uncommented and successfully tested.